src/coreclr/jit/codegenlinear.cpp

```diff
@@ -2484,8 +2484,9 @@ CodeGen::GenIntCastDesc::GenIntCastDesc(GenTreeCast* cast)
         }
 
 #if defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
-        // For LoongArch64's ISA which is same with the MIPS64 ISA, even the instructions of 32bits operation need
-        // the upper 32bits be sign-extended to 64 bits.
+        // LoongArch64 and Riscv64 ABIs require 32-bit values to be sign-extended to 64-bits.
```

Comment:
```suggestion
        // LoongArch64 and RiscV64 ABIs require 32-bit values to be sign-extended to 64-bits.
```
should it be a TODO?

Reply:
> should it be a TODO?

I think so

Reply:
Updated, so it can be queried with rest of the TODOs.